src/model/Color.ts

```diff
@@ -27,7 +27,7 @@ export class Color {
                 ModelUtils.toHexString(this.g, 2) +
                 ModelUtils.toHexString(this.b, 2);
         } else {
-            this.rgba = `rgba(${this.r},${this.g},${this.b},${this.a / 255.0})`;
+            this.rgba = `rgba(${this.r},${this.g},${this.b},${+(this.a / 255.0).toFixed(2)})`;
```

Comment:
While this is valid code in Javascript/Typescript, the C# and Kotlin compilation will fail because there is by today no `ToFixed` extension method defined:
https://github.com/CoderLine/alphaTab/runs/5294838093?check_suite_focus=true#step:6:90
https://github.com/CoderLine/alphaTab/runs/5294837733?check_suite_focus=true#step:7:113

You can learn more about the toolchain here: https://alphatab.net/docs/contributing#development-environment

You can either extend all the platforms to solve this issue (introduce extension methods) or maybe this change is not actually required and could be reverted. It's not looking very nice to have that many decimals but technically I guess it doesn't hurt.